webpack.config.js

```diff
@@ -374,7 +375,7 @@ if (IS_TEST || IS_STORYBOOK) {
 }
 
 // Dev only! Hot module reloading
-if (USE_HOT_MODULE_RELOAD) {
+if (USE_HOT_MODULE_RELOAD && !NO_DEV_SERVER) {
```

Comment:
So theoretically you could run `yarn dev-acceptance` (essentially `webpack --watch`) and `py.test -f <>` together, it's just dealing with timing between the initial `webpack` build being completed and `py.test` running that is a bit of a problem.